README.md

```diff
@@ -1,3 +1,82 @@
 # vald-client-java
 
 [![Maven Central](https://img.shields.io/maven-central/v/org.vdaas.vald/vald-client-java?style=flat-square)](https://search.maven.org/search?q=g:%22org.vdaas.vald%22%20AND%20a:%22vald-client-java%22)
+
+A Java gRPC client library for [Vald](https://github.com/vdaas/vald).
+
+## Usage
+
+### Dependency
+
+Add vald-client-java into your project dependency.
+
+For Maven `pom.xml`,
+
+```xml
+<dependency>
+    <groupId>org.vdaas.vald</groupId>
+    <artifactId>vald-client-java</artifactId>
+    <version>x.y.z</version>
+</dependency>
+<dependency>
+  <groupId>io.grpc</groupId>
+  <artifactId>grpc-netty-shaded</artifactId>
+  <version>x.y.z</version>
+</dependency>
+```
+
+Gradle `build.gradle`,
+
+```groovy
+implementation 'org.vdaas.vald:vald-client-java:x.y.z'
+implementation 'io.grpc:grpc-netty-shaded:x.y.z'
+```
+
+`io.grpc:grpc-netty-shaded` can be replaced by `io.grpc:grpc-okhttp` or `io.grpc:grpc-netty`.
+
+
+### Example
+
+```java
+import io.grpc.ManagedChannelBuilder;
+import org.vdaas.vald.api.v1.vald.InsertGrpc;
+import org.vdaas.vald.api.v1.vald.SearchGrpc;
+import org.vdaas.vald.api.v1.payload.Insert;
+import org.vdaas.vald.api.v1.payload.Search;
+import org.vdaas.vald.api.v1.payload.Object;
```

Comment:
Can we use wild card import ?

Reply:
`import org.vdaas.vald.api.v1.*` looks more simple for README

Reply:
i think it is not good to use wildcard imports here because most of the readers want to know where the classes come from.

Reply:
hmm,I think abstract imports were used for prevent dependency break, but vald uses versioned package it will never happen, so user don't need to know about Class defined package Java IDE automatically resolveit

Reply:
both of okay, if you think so it's your choice.